bitrepository-integrity-service/src/main/java/org/bitrepository/integrityservice/web/PillarDetailsDTO.java

```diff
@@ -0,0 +1,67 @@
+package org.bitrepository.integrityservice.web;
+
+import java.io.Serializable;
+
+public class PillarDetailsDTO implements Serializable {
```

Comment:
Naming: `PillarDetailsDTO` or `PillarDetailsDto`? Modern Java naming seems to prefer considering an abbreviation one word and therefore only capitalize the first letter in a class or variable name.

Reply:
I think it's a matter of taste, but honestly I also prefer only capitalizing the first letter. Will fix.